core/src/main/java/org/elasticsearch/common/blobstore/url/URLBlobStore.java

```diff
@@ -119,10 +119,10 @@ public void close() {
      * @return Base URL + path
      */
     private URL buildPath(BlobPath path) throws MalformedURLException {
-        String[] paths = path.toArray();
-        if (paths.length == 0) {
+        if (path.isEmpty) {
```

Comment:
I saw it. Just wondering how `path.isEmpty` works as I would expect `path.isEmpty()`.